utils/fo-installdeps

```diff
@@ -110,13 +110,13 @@ if [ $BUILDTIME ]; then
               lenny|hardy) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.3;;
               squeeze) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               wheezy) apt-get $YesOpt install $YesOpt postgresql-server-dev-9.1;;
-              sid) apt-get  $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               lucid|natty) apt-get $YesOpt install postgresql-server-dev-8.4;;
               isadora) apt-get $YesOpt install postgresql-server-dev-8.4;;
               precise|oneiric|quantal|raring|saucy) apt-get $YesOpt install postgresql-server-dev-9.1;;
               trusty) apt-get $YesOpt install postgresql-server-dev-9.3;;
               jessie) apt-get $YesOpt install postgresql-server-dev-9.4;;
-              xenial) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              xenial|yakkety) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              stretch|buster|sid|zesty|artful) apt-get $YesOpt install postgresql-server-dev-9.6;;
```

Comment:
it will break soon and is the same problem as in https://github.com/fossology/fossology/pull/864. Maybe it is a good idea to remove the `sid` option but I am not sure.
=> It is OK until someone finds a better solution.